build.gradle

```diff
@@ -888,7 +966,7 @@ task javadocs(type: Javadoc) {
 
     // Include names of any project that is to be included in the javadoc distribution
     ext.projects = [':client']
-    options.links("http://docs.oracle.com/javase/8/docs/api/");
+    options.links("http://docs.oracle.com/javase/10/docs/api/");
```

Comment:
We haven't made the move to Java 10 quite yet, right? Should we update this to point to the Java 10 page?

Reply:
This is just the page that will get linked to if someone clicks on the class for a return value or a parameter that is defined in the standard library (like a String). I think it is OK to link them to more current docs.

Reply:
there could be new or different behavior in the newer version, no?

Reply:
Yes, but we have no idea what version they are going to run their app with. If they run with 11 or 9 then either link is wrong. I am simply picking 10 because it is newer, common, and not past end of life.